stumpy/core.py

```diff
@@ -577,6 +577,8 @@ def _welford_nanvar(a, w, a_subseq_isfinite):
                 * (a[last_idx] - curr_mean + a[prev_start_idx] - prev_mean)
                 / w
             )
+            if curr_var < 1e-8:
```

Comment:
>  why 1e-8 and, say, not 1e-14?

No particular reason to be honest. So, I have been trying different values. It seems  whether this threshold is good or not depends on the scale of values. I now set it to `1e-5`! I also "almost" avoid using `STUMPY_STDDEV_THRESHOLD`. I mean...I set it now to 1e-100. 

I think these two config variables should be treated differently:

* `STUMPY_STDDEV_THRESHOLD` tell us if we should treat the "std value" as `0` or not.
* `STUMPY_VAR_THRESHOLD`  tells us if we need to re-calculate the variance (because if it is small, we prefer to avoid imprecision as its impact can be catastrophic)

Reply:
> So, I am not sure if we should apply the equation var = std^2 to these thresholds

Got it. So maybe it should be called something like `STUMPY_MIN_VAR_PRECISION` instead?

Reply:
Yeah...I think this name fits better.